docs/en/observability/uptime-lightweight-monitors.asciidoc

```diff
@@ -0,0 +1,35 @@
+[[uptime-lightweight-monitors]]
+= Use lightweight monitors
+
+// What do we want to say about lightweight monitors?
+
+In the {uptime-app}, you can monitor the status of network endpoints using the following lightweight checks:
+
+// lint ignore v4 v6
+|===
+| *HTTP monitor* | Monitor your website. The HTTP monitor checks to make sure specific endpoints return the correct
+status code and display the correct text.
+| *ICMP monitor* | Check the availability of your hosts. The ICMP monitor uses ICMP (v4 and v6) Echo
+Requests to check the network reachability of the hosts you are pinging. This will tell you whether the
+host is available and connected to the network, but doesn't tell you if a service on the host is running or
+not.
+| *TCP monitor* | Monitor the services running on your hosts. The TCP monitor checks individual ports
+to make sure the service is accessible and running.
+|===
+
+// diagram similar to https://www.elastic.co/guide/en/observability/current/synthetics-journeys.html ? 
+
+[discrete]
+== Configure a monitor
+
+// List all options?
+
+[discrete]
+== Create a monitor
+
+// Options are available in the set up guide
+
+[discrete]
+== Manage monitors
+
+// How to 
```

Comment:
Not sure I understand what you mean by manage. Is it editing and deleting them? Or is it managing the results (i.e. viewing results and debugging)?

Reply:
Meant to be the lightweight equivalent of [Manage browser monitors](https://observability-docs_2073.docs-preview.app.elstc.co/guide/en/observability/master/synthetics-manage-monitors.html), but if there's not enough to say specifically for lightweight monitors (like there is for browser monitors) then we can remove it.